tests/test_warp.py

```diff
@@ -54,7 +54,7 @@ def flatten_coords(coordinates):
                 yield x
 
 
-if gdal_version.at_least("3.7"):
+if gdal_version.at_least("3.6"):
```

Comment:
This change occurs in 3.6.1. However since 3.6.0 has been redacted [ref](https://github.com/OSGeo/gdal/releases/tag/v3.6.1), this should be a safe change to make.